fns-hl7-pipeline/fn-receiver-debatcher/src/main/kotlin/gov/cdc/dex/hl7/Function.kt

```diff
@@ -193,8 +193,13 @@ class Function {
             startTime = startTime,
             errorMessage = errorMessage
         )
-        val receiverEventError = ReceiverEventError(messageIndex,messageMetadata.messageUUID, errorMessage)
-        eventReport.errorMessages = mutableListOf(receiverEventError)
+
+        if(!errorMessage.isNullOrEmpty()) {
+            val receiverEventError = ReceiverEventError(messageIndex, messageMetadata.messageUUID, errorMessage)
+            eventReport.errorMessages = mutableListOf(receiverEventError)
```

Comment:
You are replacing the list every time ? Should add to the list.
There is a function here, addErrorToReport. Why not use it?

Reply:
Done.